### PR TITLE
create-product-detail

### PR DESCRIPTION
**Pull request by canvantuan12**

### CODE REVIEW

File: app/controllers/admin/products_controller.rb [hunk 1]
```diff
@@ -19,13 +20,37 @@ def create
     end
   end
 
+  def destroy
+    if @product.destroy
+      flash[:success] = t ".destroysuccess"
```
Comment (nhunght) [inline app/controllers/admin/products_controller.rb]: destroy_success

File: app/controllers/admin/products_controller.rb [hunk 2]
```diff
@@ -19,13 +20,37 @@ def create
...
+      redirect_to :back
+    else
+      flash[:danger] = t ".destroyfail"
```
Comment (nhunght) [inline app/controllers/admin/products_controller.rb]: destroy_fail

File: app/controllers/admin/products_controller.rb [hunk 3]
```diff
@@ -19,13 +20,37 @@ def create
...
+      redirect_to :back
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if @product.update_attributes product_params
+      flash[:success] = t ".updatesuccess"
```
Comment (nhunght) [inline app/controllers/admin/products_controller.rb]: update_success

File: app/controllers/admin/products_controller.rb [hunk 4]
```diff
@@ -19,13 +20,37 @@ def create
...
+      redirect_to admin_products_path
+    else
+      flash[:danger] = t ".updatefail"
```
Comment (nhunght) [inline app/controllers/admin/products_controller.rb]: update_fail

Comment (hathanhtamnd) [inline app/controllers/admin/products_controller.rb]: update_fail

File: app/controllers/admin/products_controller.rb [hunk 5]
```diff
@@ -19,13 +20,37 @@ def create
...
+
   private
   def load_categories
     @categories = Category.with_children
   end
 
   def product_params
-    params.require(:product).permit :category_id, :image, :product_name,
-      :price, :description, :quantity
+    params.require(:product).permit(:category_id, :image, :product_name,
```
Comment (hathanhtamnd) [inline app/controllers/admin/products_controller.rb]: Bỏ ngoặc

**Pull request merged**